terraform/dotnet/ec2/windows/traffic-generator-setup.ps1

```diff
@@ -0,0 +1,32 @@
+param (
+    [string]$RemoteServicePrivateEndpoint,
+    [string]$TestID,
+    [string]$TestCanaryType
+)
+
+wget -O nodejs.zip https://nodejs.org/dist/v20.16.0/node-v20.16.0-win-x64.zip
+Expand-Archive -Path .\nodejs.zip -DestinationPath .\nodejs -Force
+$currentdir = Get-Location
+Write-Host $currentdir
+$env:Path += ";$currentdir" + "\nodejs\node-v20.16.0-win-x64"
+
+# Bring in the traffic generator files to EC2 Instance
+#aws s3 cp "s3://aws-appsignals-sample-app-prod-${var.aws_region}/traffic-generator.zip" "./traffic-generator.zip"
```

Comment:
A lot of leftover comments it looks like, let's remove them not just here but in all files.